packages/react/src/QuestionnaireForm/QuestionnaireFormItem/QuestionnaireFormItem.tsx

```diff
@@ -436,6 +472,14 @@ function isDropDownChoice(item: QuestionnaireItem): boolean {
   );
 }
 
+function isMultiSelectChoice(item: QuestionnaireItem): boolean {
+  return !!item.extension?.some(
```

Comment:
The `getExtensionValue` helper could be useful here.

Reply:
was doing some testing but left as is to keep it consistent with the rest of the file